celery_batches/__init__.py

```diff
@@ -342,6 +342,7 @@ def _do_flush(self) -> None:
         if len(ready_requests) > 0:
             logger.debug("Batches: Ready buffer complete: %s", len(ready_requests))
             self.flush(ready_requests)
+            self._count = count(1)
```

Comment:
Is resetting to 1 correct? Should it reset to what is left in `_pending`?

I tried to play with that a bit, but wasn't able to come up with a good test case.